analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -360,10 +362,11 @@ open class Npm(
         return Pair(id.toCoordinates(), module)
     }
 
-    protected open fun getRemotePackageDetails(workingDir: File, packageName: String): JsonNode {
-        val process = run(workingDir, "view", "--json", packageName)
-        return jsonMapper.readTree(process.stdout)
-    }
+    protected open fun getRemotePackageDetails(workingDir: File, packageName: String): JsonNode =
```

Comment:
This function might be called in parallel via `parsePackage`, so should `npmViewCache` become a `ConcurrentHashMap` like `submodulesCache` below?